lib/src/asset/cache.dart

```diff
@@ -58,27 +58,41 @@ class CachedAssetReader extends AssetReader {
 
   @override
   Future<bool> hasInput(AssetId id) {
```

Comment:
Add comment as to why this is not `async` – it makes sense now, but docs would help.